code/src/nuvla/connector/nuvlabox_connector.py

```diff
@@ -204,7 +204,7 @@ def update_nuvlabox_engine(self, **kwargs):
 
         nb_status = self.get_nuvlabox_status()
         if 'installation-parameters' not in nb_status:
-            raise Exception(f'Installation parameters are required, but are not present in NuvlaBox status {nb_status.id}')
+            logging.warning(f'Installation parameters are required, but are not present in NuvlaBox status {nb_status.id}')
```

Comment:
I had an exception in the generation of the error message because `nb_status` is a dict and has no attribute `id`.
```python
  File \"/usr/local/lib/python3.9/site-packages/nuvla/connector/nuvlabox_connector.py\", line 207, in update_nuvlabox_engine
    raise Exception(f'Installation parameters are required, but are not present in NuvlaBox status {nb_status.id}')
AttributeError: 'dict' object has no attribute 'id'
```